clientv3/integration/black_hole_test.go

```diff
@@ -26,6 +27,96 @@ import (
 	"github.com/coreos/etcd/pkg/testutil"
 )
 
+// TestBalancerUnderBlackholeWatch ensures that watch client
+// switch its endpoints when the member of the pinned endpoint is blackholed.
+func TestBalancerUnderBlackholeWatch(t *testing.T) {
```

Comment:
what is the difference between this one and `TestWatchKeepAlive`?

Reply:
Yeah, `TestWatchKeepAlive` should cover this case. Closing. Thanks!

Reply:
let us rename that test to make everything consistent.